File: plan/leetcode/LCR_135.py


from typing import List


class Solution:
    def countNumbers(self, cnt: int) -> List[int]:
        m = 10 ** cnt
        buf = [i for i in range(1, m)]
        return buf
        

def main():
    print(Solution().countNumbers(2))

if __name__ == "__main__":
    main()

File: plan/leetcode/LCR_135.go
package leetcode

import (
	"math"
)

func countNumbers(cnt int) (buf []int) {

	m := math.Pow(10, float64(cnt))

	// traverse
	buf = make([]int, int(m)-1)
	for i := 1; i < int(m); i++ {
		buf[i-1] = i
	}
	return buf
}
